Source/Utils/WarpXMovingWindow.cpp

```diff
@@ -44,7 +44,7 @@ WarpX::MoveWindow (bool move_j)
 
     // Update the continuous position of the moving window,
     // and of the plasma injection
-    moving_window_x += moving_window_v * dt[0];
+    moving_window_x += (moving_window_v - WarpX::beta_boost * PhysConst::c)/(1 - moving_window_v * WarpX::beta_boost / PhysConst::c) * dt[0];
```

Comment:
I am afraid this change here does not look correct. Can you elaborate a little what you are aiming for to achieve?